Dockerfile

```diff
@@ -1,25 +1,28 @@
 # Copyright (c) 2022 LG Electronics Inc.
 # SPDX-License-Identifier: Apache-2.0
-FROM	ubuntu:20.04
+FROM amd64/ubuntu:20.04
```

Comment:
Consider using `python` instead of `ubuntu` for your base image, `ubuntu` images can be large because they contain other packages including python.